static/app/components/group/assignedTo.tsx

```diff
@@ -112,7 +113,7 @@ const StyledSidebarSectionContent = styled(SidebarSection.Content)`
 `;
 
 const StyledLoadingIndicator = styled(LoadingIndicator)`
-  height: 20px;
-  width: 20px;
-  margin: 0;
+  width: 24px;
+  height: 24px;
+  margin: 0 !important;
```

Comment:
global styles :(

Reply:
you could also use `&& {}` here. I usually use it just to have the feeling that I am not using `!important` 😄